docs/orchestrating-elastic-stack-applications/security/custom-http-certificate.asciidoc

```diff
@@ -51,4 +51,68 @@ spec:
     kind: Issuer
     name: selfsigned-issuer
   secretName: quickstart-es-cert
+  subject:
+    organizations:
+      - quickstart
+----
+
+
+Here is another example to issue multiple certificates from the same self-signed CA:
```

Comment:
I think this need a bit more explanation so that readers better understand what we are trying to achieve here and what the benefits are compared to the first example.